src/Cooked/MockChain/Testing.hs

```diff
@@ -271,6 +104,8 @@ instance IsProp QC.Property where
 forAll :: (Show a) => QC.Gen a -> (a -> QC.Property) -> QC.Property
 forAll = QC.forAll
 
+-- | Bool instance of 'IsProp' instances
+
 -- TODO: Discuss this instance; its here to enable us to easily run things in a
 -- repl but I'm not sure whether to ignore the counterexample messages or not.
```

Comment:
How about this old TODO? What was the context?

Reply:
Good point as well. I have no idea whether we ever used the Boolean instance TBH. Maybe it should go.

Reply:
I think it should. Worst case scenario we resurrect it.